src/maintenance/repositories/official/nonfree.md

```diff
@@ -0,0 +1,11 @@
+# nonfree
+
+Void has a `nonfree` repository for packages that don't have free licenses.
+Install the `void-repo-nonfree` package to enable this repository.
+
+Packages can end up in the `nonfree` repository for a number of reasons:
```

Comment:
This is fine for now, but it would be really nice to have a policy document for what winds up in what repo.  Maybe @maxice8  would be willing to shepherd a PR through to void-packages for repository policy.

Reply:
You guys figure this out.

Reply:
I didn't mean to be rude here, and I'm willing to help with this, but it is way above my head specifying how packages end up in each repo!